### ISSUE TITLE
ngrok 2.0 token

### ISSUE THREAD

**Issue by attiks** [bubenshchykov/ngrok#27]
I played with the 2.0 branch and it works great, only thing is you need a token.

Can you add to the readme that people can install the token using 

```
./ngrok authtoken THE_TOKEN
```
or 
```
./node_modules/ngrok/bin/ngrok authtoken THE_TOKEN
```

it will save the token in a yml file, that gets included automatically, so you no longer need to specify the token inside the nodejs file, avoiding it might get committed.

**Comment by bubenshchykov** [comment]
@attiks good point, I'll update the readme when I'm about to publish 2.0 to npm. currently I'm waiting for one ngrok fix, it comes in a week or two. till that I actually recommend using latest version published on npm, it's stable and great too :)

**Comment by attiks** [comment]
The problem is I need to be able to use ```host_header``` and it didn't worked with the latest npm version, unless I missed something.

Out of curiosity, what is the problem with ngrox 2, it is working great

**Comment by bubenshchykov** [comment]
ngrok on npm uses 1.x version, it will be 2.0 in few weeks

current master of node wrapper for ngrok uses ngrok 2.0, actively uses ngrok 2.0 api for speed wins. the bug is that ngrok 2.0 designed we a thought that api use requires authtoken. it's a bug and Allan (ngrok owner) will fix it. The tests of master branch with ngrok 2.0 are green  only because of this hack 
https://github.com/bubenshchykov/ngrok/blob/master/test/chai.js#L5

I can't release it to npm now because it will break existing users. So let's wait when authtoken dependency is fixed, gonna be cool :)

**Comment by cardoni** [comment]
:+1:

**Comment by TrySound** [comment]
Hi, @bubenshchykov! Any news?

**Comment by BrandonZacharie** [comment]
+1

**Comment by bubenshchykov** [comment]
@inconshreveable Hi Alan! Is authtoken still required for ngrok api use?

**Comment by achingbrain** [comment]
> I can't release it to npm now because it will break existing users who have no authtoken

Could you not release it with a major version bump?  Semver says that indicates a non-backwards compatible change..

**Comment by BrandonZacharie** [comment]
+1

**Comment by inconshreveable** [comment]
Still working on this. :( Sorry for the insanely long delay here.

**Comment by bubenshchykov** [comment]
guys I've published to ngrok@0.2.0 to npm and it uses latest ngrok 2. better later than never :)

also added info about mandatory authtoken to description and made a separate issue to make authtoken optional like it is in ngrok@0.1.99 https://github.com/bubenshchykov/ngrok/issues/33

**Comment by TrySound** [comment]
@bubenshchykov Yupee!
